en/docs/administer/logging-and-monitoring/logging/configuring-logging.md

```diff
@@ -218,7 +218,12 @@ The following is a sample Gateway Wire Log for an API request.
      loggers = synapse-headers, trace-messages, org-apache-coyote,com-hazelcast
      ```
 
-4.  Observe the logs for incoming and outgoing traffic in the `<API-M_HOME>/repository/logs/wso2carbon.log` file.
+4. Observe the logs for incoming and outgoing traffic in the `<API-M_HOME>/repository/logs/wso2carbon.log` file.
+
+!!! note
```

Comment:
!!! Limitation

If synapse-wire logs are enabled *during high load*, outgoing traffic between the gateway and the backend will not be printed in the `wso2carbon.log` file. This happens when the backend connection is keep alive. As we are maintaining connections in a connection pool, the same connections will be reused in keep alive mode irrespective of the changes done in the `log4j2.properties file`.

Reply:
Shall we reword this limitation like above?